xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,402 @@
+
+        _update_modified_time = self._init_patch_with_name('_update_modified_time', 'xmind.core.sheet.SheetElement.updateModifiedTime', )
+
+        _rel = Mock()
+        _rel.getImplementation.return_value = "test"
+
+        _obj = SheetElement()
+
+        self.assertEqual(_obj.removeRelationship(_rel), None)
+        _get_relationships.assert_called_once()
+        _rel.getImplementation.assert_called_once()
+        _rels.removeChild.assert_called_once_with("test")
+        _rels.hasChildNodes.assert_called_once()
+        _get_implementation.assert_called_once()
+        _get_implementation_obj.removeChild.assert_called_once_with(_rels)
+        _update_modified_time.assert_called_once()
+
+    def test_remove_relationship_with_set_up_relationships_and_with_child_nodes(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic')
+        _rels = Mock()
+        _rels.removeChild.return_value = None
+        _rels.hasChildNodes.return_value = Mock()
+        _get_relationships = self._init_patch_with_name('_get_relationships', 'xmind.core.sheet.SheetElement._getRelationships', return_value=_rels)
+
+        _get_implementation_obj = Mock()
+        _get_implementation_obj.removeChild.return_value = None
+        _get_implementation = self._init_patch_with_name('_get_implementation', 'xmind.core.sheet.SheetElement.getImplementation', return_value=_get_implementation_obj)
+
+        _update_modified_time = self._init_patch_with_name('_update_modified_time', 'xmind.core.sheet.SheetElement.updateModifiedTime', )
+
+        _rel = Mock()
+        _rel.getImplementation.return_value = "test"
+
+        _obj = SheetElement()
+
+        self.assertEqual(_obj.removeRelationship(_rel), None)
+        _get_relationships.assert_called_once()
+        _rel.getImplementation.assert_called_once()
+        _rels.removeChild.assert_called_once_with("test")
+        _rels.hasChildNodes.assert_called_once()
+        _get_implementation.assert_not_called()
+        _get_implementation_obj.removeChild.assert_not_called()
+        _update_modified_time.assert_called_once()
+
+    def test_get_root_topic(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic', return_value="RootTopic")
+
+        _obj = SheetElement()
+
+        self.assertEqual(_obj.getRootTopic(), "RootTopic")
+
+    def test_get_title(self):
+        _get_root_topic = self._init_patch_with_name('_get_root_topic', 'xmind.core.sheet.SheetElement._get_root_topic')
+        _get_first_child_node_by_tag_name = self._init_patch_with_name('_get_first_child_node_by_tag_name', 'xmind.core.sheet.SheetElement.getFirstChildNodeByTagName')
+
+        _obj = SheetElement()
+
+        self.assertEqual(_obj._get_title(), None)
+        _get_first_child_node_by_tag_name.assert_called_once_with(TAG_TITLE)
+
+    def test_class_method_get_title_without_title(self):
```

Comment:
```test_class_method_``` naming doesn't follow convention even in one file.

Reply:
It caused by existence two methods: _get_title and getTitle

Reply:
so, then rename you tests in the next convention 
test_<ORIGINAL_NAME_OF_THE_METHOD>_whatever_you_chek_in_it
then you will not have any problems